index.html

```diff
@@ -37,6 +37,148 @@ <h1>Hey there. I'm <br />Subrat </h1>
         <i class="fa fa-chevron-down"></i>
       </span>
     </section>
+    <section id="Works">
+      <h1 class="sectionHeading">My Recent <br />Works </h1>
+      <div class="bar"></div>
+      <div class="card">
+        <div class="cardImage">
 
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>

```

Comment:
- [ ] Kindly wrap an `a` tag inside each `li` tag. In a real website, each badge would link to a site when clicked. That is the same idea used when building your portfolio. Kindly implement this change 🙏 
You can implement it by doing this `<li><a>Ruby on rails</a></li>`

Reply:
updated